pyproject.toml

```diff
@@ -7,23 +7,21 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.10"
+langchain = "^0.1.5"
+langchain-openai = "^0.0.3"
+langchain-google-genai = "^0.0.6"
+langserve = {extras = ["server"], version = "^0.0.39"}
 fastapi = "^0.103.1"
 pydantic = "1.10"
-langchain = { git = "https://github.com/langchain-ai/langchain.git", subdirectory = "libs/langchain" }
 uvicorn = "^0.23.2"
 beautifulsoup4 = "^4.12.2"
 weaviate-client = "^3.23.2"
 psycopg2 = "^2.9.7"
 lxml = "^4.9.3"
-langserve = {extras = ["server"], version = "^0.0.39"}
 voyageai = "^0.1.4"
 anthropic = "^0.10.0"
 pillow = "^10.2.0"
 tiktoken = "^0.5.2"
-langchain-community = "^0.0.14"
-langchain-openai = "^0.0.3"
-langchain-google-genai = "^0.0.6"
-langchain-core = "0.1.15-rc.2"
```

Comment:
Should we pin a later version of langchain core or will that be picked up by default?